src/main/resources/migrations/04-13-2022_add_help_xp_tables.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE help_account (
+  user_id BIGINT PRIMARY KEY,
+  experience DOUBLE NOT NULL,
+);
+
+CREATE TABLE help_transaction (
+  id BIGINT PRIMARY KEY AUTO_INCREMENT,
+  recipient BIGINT NOT NULL,
+  created_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6),
+  value DOUBLE NOT NULL,
+  message VARCHAR(255)
```

Comment:
I am quite unsure if we really want to store up to 255 characters per transaction as there would be a pretty high amount of those.